chromium_src/BUILD.gn

```diff
@@ -1,10 +1,17 @@
 if (is_win) {
-  source_set("google_update") {
```

Comment:
@simonhong, what is the purpose of this source_set? Seems like we use a patch on //google_update/BUILD.gn to use this .idl file. (Disregard my change - that's just to pacify GN).

Reply:
Oh.. I should've erased it before. This target is not used.

Reply:
Cool, I'll remove it.